tests/gdx-tests/src/com/badlogic/gdx/tests/g3d/PolarAccelerationTest.java

```diff
@@ -0,0 +1,180 @@
+
+	@Override
+	public void create () {
+		super.create();
+		emitters = new Array<ParticleController>();
+		assets.load(DEFAULT_PARTICLE, Texture.class);
+		assets.load(DEFAULT_SKIN, Skin.class);
+		loading = true;
+		environment = new Environment();
+		billboardParticleBatch = new BillboardParticleBatch();
+		billboardParticleBatch.setCamera(cam);
+		ui = new Stage();
+		builder = new StringBuilder();
+	}
+
+	@Override
+	public void resize (int width, int height) {
+		super.resize(width, height);
+		ui.getViewport().setWorldSize(width, height);
+		ui.getViewport().update(width, height, true);
+	}
+
+	@Override
+	protected void onLoaded () {
+		Texture particleTexture = assets.get(DEFAULT_PARTICLE);
+		billboardParticleBatch.setTexture(assets.get(DEFAULT_PARTICLE, Texture.class));
+		addEmitter(particleTexture);
+		setupUI();
+	}
+
+	private void addEmitter (Texture particleTexture) {
+		ParticleController controller = createBillboardController(particleTexture);
+		controller.init();
+		controller.start();
+		emitters.add(controller);
+
+		controller.translate(new Vector3(5, 0, 5));
+		controller.rotate(new Vector3(.707f, .707f, 0), 135);
+	}
+
+	private void setupUI () {
+		Skin skin = assets.get(DEFAULT_SKIN);
+		Table table = new Table();
+		table.setFillParent(true);
+		table.top().left().add(new Label("FPS ", skin)).left();
+		table.add(fpsLabel = new Label("", skin)).left().expandX().row();
+		ui.addActor(table);
+	}
+
+	private ParticleController createBillboardController (Texture particleTexture) {
+		// Emission
+		RegularEmitter emitter = new RegularEmitter();
+		emitter.getDuration().setLow(3000);
+		emitter.getEmission().setHigh(300);
+		emitter.getLife().setHigh(4000);
+		emitter.setMaxParticleCount(3000);
+
+		// Spawn
+
+		CylinderSpawnShapeValue cylinderSpawnShapeValue = new CylinderSpawnShapeValue();
+		cylinderSpawnShapeValue.spawnWidthValue.setHigh(5);// x
+		cylinderSpawnShapeValue.spawnHeightValue.setHigh(10);// y
+		cylinderSpawnShapeValue.spawnDepthValue.setHigh(5);// z
+		cylinderSpawnShapeValue.setEdges(true);
+
+		SpawnInfluencer spawnSource = new SpawnInfluencer(cylinderSpawnShapeValue);
+		// Dynamics
+		DynamicsInfluencer dynamicsInfluencer = new DynamicsInfluencer();
+
+		PolarAcceleration polarAcceleration = new PolarAcceleration();
+		polarAcceleration.thetaValue.setActive(true);
+		polarAcceleration.thetaValue.setTimeline(new float[] {0});
+		polarAcceleration.thetaValue.setScaling(new float[] {1});
+		polarAcceleration.thetaValue.setHigh(90);
+		polarAcceleration.phiValue.setActive(true);
+		polarAcceleration.phiValue.setTimeline(new float[] {0});
+		polarAcceleration.phiValue.setScaling(new float[] {1});
+		polarAcceleration.phiValue.setHigh(0);
+		polarAcceleration.strengthValue.setActive(true);
+		polarAcceleration.strengthValue.setHigh(10);
+		polarAcceleration.strengthValue.setTimeline(new float[] {0});
+		polarAcceleration.strengthValue.setScaling(new float[] {1});
+		polarAcceleration.isGlobal = false;
```

Comment:
Running the test in this file demonstrates the correct behavior when the PolarAcceleration's direction is properly updated and exhibits incorrect behavior without that change. If isGlobal is set to true here, the old behavior is retained, so changing this to true and running the test demonstrates the old behavior that's being corrected.